frontend/src/views/Enrollment/EnrollmentForm.js

```diff
@@ -0,0 +1,310 @@
+                    Your manager&rsquo;s name
+                    (<InfoLink page={MENTORING_MANA_PAGE}>how we use this</InfoLink>)
+                  </Fragment>
+                )} />
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                {...textFieldProps('manager_email')}
+                helperText="Your manager's email" />
+            </Grid>
+            <Grid item xs={12}>
+              <AvailabilitySelector timeAvailability={participant.time_availability} onChange={handleTimeAvailabilityChange} />
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                {...textFieldProps('org')}
+                label="Organization"
+                placeholder="Firefox"
+                helperText="Your Organization at Mozilla" />
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                {...textFieldProps('org_level')}
+                label="Organization Level"
+                placeholder="IC3"
+                helperText={(
+                  <Fragment>
+                    Your Organization Level
+                    (<InfoLink page={CLG_PAGE}>career leveling guides</InfoLink>)
+                  </Fragment>
+                )} />
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                {...textFieldProps('time_at_org_level')}
+                label="Time at Level"
+                select
+                helperText="Time you have been at this organizational level" >
+                {menuItems(TIMES_AT_ORG_LEVEL)}
+              </TextField>
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                {...textFieldProps('org_chart_distance')}
+                label="Organizational Distance"
+                select
+                helperText="Would you like to be paired with someone close to you on the Organization Chart?" >
+                {menuItems(ORG_CHART_DISTANCES)}
+              </TextField>
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                {...textFieldProps('track_change')}
+                label="Track Change"
+                disabled={!learner}
+                required={learner}
+                select
+                helperText="Are you considering change track (such as between IC and management)?" >
+                {menuItems(TRACK_CHANGE_INTEREST)}
+              </TextField>
+            </Grid>
+            <Grid item sm={6} />
+            <Grid item xs={12} sm={6}>
+              <InterestsControl
+                title="Learner Interests"
+                subheader="Select the topics where you are interested in improving"
+                disabled={!learner}
+                choices={CANNED_INTERESTS}
+                interests={participant.learner_interests}
+                onChange={setLearnerInterests} />
```

Comment:
ah, I think it'd be less confusing to call the `onChange` prop`setLearnerInterests`, since `onChange` is a built-in/inline event handler.

Reply:
Maybe onInterestChange?  It's only _learner_ interests in this use of the component -- just below it's mentor interests.

I chose "onXxx" here (and on the AvailabilitySelector, and on the EnrollmentForm itself) because this is essentially a controlled component.  I can't actually find any best-practices for building such things, though.

Reply:
yeah, I think having a more descriptive `on<something>Change` would be clearer.

Reply:
Awesome - thanks for the help!